zerver/lib/external_accounts.py

```diff
@@ -10,12 +10,16 @@
 
 DEFAULT_EXTERNAL_ACCOUNTS = {
     "twitter": {
-        "text": "Twitter",
-        "url_pattern": "https://twitter.com/%(username)s"
+        "text": "Twitter",  # Realm admin sees in field options
+        "url_pattern": "https://twitter.com/%(username)s",
+        "name": "Twitter",  # Field label or name
+        "hint": "Only your username",
     },
     "github": {
         "text": 'GitHub',
-        "url_pattern": "https://github.com/%(username)s"
+        "url_pattern": "https://github.com/%(username)s",
+        "name": "GitHub",
+        "hint": "Only your GitHub username",
```

Comment:
@rishig what should the hint be here?  "Enter your GitHub username" sounds good to me.

Reply:
yeah, sgtm